pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,434 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <%- include('../partials/head'); %>
+  <link href="/stylesheets/theme.bootstrap.css" rel="stylesheet" />
+  <script src="/javascripts/jquery.tablesorter.min.js"></script>
+  <script src="/javascripts/jquery.tablesorter.widgets.js"></script>
+</head>
+
+<body>
+  <script>
+    $(function () {
+      $('[data-toggle="popover"]').popover({ sanitize: false })
+    });
+
+    // make the file inputs display the file name
+    $(document).on('change', '.custom-file-input', function () {
+      let fileName = $(this).val().replace(/\\/g, '/').replace(/.*\//, '');
+      $(this).parent('.custom-file').find('.custom-file-label').text(fileName);
+    });
+  </script>
+  <%- include('../partials/navbar'); %>
+  <div id="content" class="container-fluid">
+
+    <% if(typeof errormsg !== 'undefined' && errormsg.length > 0) { %>
+      <div class="alert alert-danger" role="alert">
+        <%= errormsg %>
+      </div>
+    <% } %>
+
+    <% if (!isgroup) { %>
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white d-flex align-items-center">
+        <%= assessment_set.name %> <%= assessment.number %>: Groups
+      </div>
+      <div class="card-body">
+        This is not a group assessment. To enable this function, please set groupWork to be true in infoAssessment.json.
+      </div>
+    </div>
+
+    <% } else { %>
+    <div id="content" class="container-fluid">
+      <div class="modal fade" id="uploadAssessmentGroupsModal" tabindex="-1" role="dialog"
+        aria-labelledby="uploadAssessmentGroupsModalLabel">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h1 class="h4 modal-title" id="uploadAssessmentGroupsModalLabel">Upload new group setting</h1>
+            </div>
+            <div class="modal-body">
+              <p>
+                Upload a CSV file in the format of:
+              </p>
+            </div>
+            <div class="container bg-light py-2 border-bottom border-top">
```

Comment:
This should be under the `modal-body` container for consistent styling (we want padding on all sides of the modal body).